fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/AbstractEventCreationFragment.java

```diff
@@ -55,33 +57,80 @@ public abstract class AbstractEventCreationFragment extends Fragment {
   private EditText startTimeEditText;
   private EditText endTimeEditText;
 
+  private boolean toBeCleared = true;
```

Comment:
I have added a suggested time to the events. When the user want to change it, it would be cumbersome without clearing the suggested fields. So on the first click on one of those fields I clear them all. This variable is there to prevent the clearing on each click.
I agree it should be documented